source/math/FloatVectorList.ooc

```diff
@@ -432,9 +433,51 @@ FloatVectorList: class extends VectorList<Float> {
 		}
 		(crossCorrelation, offsets)
 	}
+	// Assumes equi-distant samples
+	// https://en.wikipedia.org/wiki/Spline_interpolation
+	interpolateSpline: func (numberOfPoints: Int) -> This {
+		result := This new(this count + numberOfPoints * (this count - 1))
+		thisPointer := this pointer as Float*
+		coefficientMatrix := This _generateCubicSplineCoefficients(this count)
+		b := FloatMatrix new(1, this count) take()
```

Comment:
I don't like `b` generally, and especially not since you use another `b` just a few lines down. `rightHandSide` is also fine.